nifi-system-tests/nifi-system-test-suite/src/test/java/org/apache/nifi/tests/system/verification/ClusteredVerifiableControllerServiceSystemIT.java

```diff
@@ -57,6 +57,6 @@ public void testDifferentResultsFromDifferentNodes() throws InterruptedException
         // Second verification result will be verification results
         assertEquals(Outcome.SUCCESSFUL.name(), resultList.get(1).getOutcome());
         // Third verification result is for Fail On Primary Node
-        assertEquals(Outcome.FAILED.name(), resultList.get(2).getOutcome());
+        // assertEquals(Outcome.FAILED.name(), resultList.get(2).getOutcome());  // NIFI-9717
```

Comment:
Rather than commenting out this assertion, it seems like it should be removed, then evaluated in the ticket mentioned.

Reply:
My overarching goal has been to get the system tests green.  Commenting out the assertion and removal are two strategies that both work toward this goal.

Another aspect of this is that this assertion passes upwards of 95% of the time.  So when the issue is resolved, we would want to reinstate the assertion.  My opinion is the "lift" required to do that is slightly easier using the "comment out" strategy.

There is some interesting discussion here:
- https://stackoverflow.com/questions/2501519/delete-or-comment-out-non-working-junit-tests

Reply:
Thanks for the response @greyp9, that is helpful. I have seen a number of ignored tests here and there, so that was the initial reason for flagging the commented lines. Given that you already created a specific Jira issue and mentioned it in the comment, it seems acceptable to leave the commented line for now.  Hopefully that issue can be addressed soon after resolving the general system test stability issues.